aixtend/modules/pipeline.py

```diff
@@ -160,7 +161,14 @@ def run_async(self, data: str, name: str = "pipeline_process"):
             'x-api-key': self.api_key,
             'Content-Type': 'application/json'
         }
-        payload = json.dumps({ "data": data })
+        if type(data) == dict:
+            payload = json.dumps(data)
+        else:
+            try:
+                data_json = json.loads(data)
+                payload = data
+            except:
+                payload = json.dumps({ "data": data })
```

Comment:
Should we accept a string formatted json in the first place?

Reply:
I am just addressing the case since it can happen.